pyproject.toml

```diff
@@ -171,6 +171,8 @@ rich = "~12.5.1"
 towncrier = "~22.8.0"
 # Watch files for celery and mkdocs reload
 watchdog = "~2.1.9"
+# Store tests results in XML format
+unittest-xml-reporting = "~3.2.0"
```

Comment:
Looking at the pypi page for this library, it has this caveat:

>unittest has the concept of sub-tests for a unittest.TestCase; this doesn't map well to an existing xUnit concept, so you won't find it in the schema. What that means, is that you lose some granularity in the reports for sub-tests.
>
> unittest also does not report successful sub-tests, so the accounting won't be exact.

We do use subTest in a number of our tests, does this pose any problems/concerns?